### DISCUSSION TITLE
Synthesize Chinese, the sound of the last kanji lasts for a long time

### DISCUSSION THREAD

**Discussion by twang5012** [coqui-ai/TTS#3049] [Qs for Live Q&A]
tts.tts_to_file(text="北京欢迎你。不到长城非好汉 北京是六朝古都", file_path="c3.wav") 
the pronounce of ‘北京是六朝古都’  repeat in c3.wav

**Comment by wjnodejs** [comment]
tts.tts_to_file(text="北京欢迎你。不到长城非好汉 北京是六朝古都。", file_path="c3.wav")，need 。at the end

**Comment by DeissVem** [comment]
注意下标点试试

**Reply by damonlid** [reply]
你好，可以请教几个关于tts 中文模型的问题吗？有哪些已经训练好的中文模型的效果较好？如何调优？望回复

**Comment by Liujingxiu23** [comment]
hi  求教下，你阅读了相关代码了吗，我不太明白，为什么推理过程分为一下两步

gpt_codes = gpt_inference.generate(text, cond_latents)
gpt_latents = gpt(text, gpt_codes,cond_latents)

为什么先GPT生成离散的 codes，再跑一次GPT生成离线的float的 latents

相关问题我也放在这里了 https://github.com/coqui-ai/TTS/discussions/3223

**Reply by OnceJune** [reply]
This design comes from Tortoise, the author indicates latents got richer semantic info than codes
